ui/apps/platform/src/Containers/SystemHealth/UsageStatistics/ShowProductUsage.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { ReactElement, useState } from 'react';
+import { Button, ButtonVariant, Modal, ModalBoxBody, ModalVariant } from '@patternfly/react-core';
+import UsageStatisticsForm from './UsageStatisticsForm';
+
+function ShowProductUsage(): ReactElement {
+    const [isModalOpen, setIsModalOpen] = useState(false);
+
+    function onCloseModalHandler() {
+        setIsModalOpen(false);
+    }
+
+    return (
+        <>
+            <Button
+                key="open-select-modal"
+                data-testid="product-usage-modal-open-button"
+                variant={ButtonVariant.secondary}
+                onClick={() => {
+                    setIsModalOpen(true);
+                }}
+            >
+                Show product usage
+            </Button>
+            <Modal
+                title="Product usage"
+                description="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin blandit augue est. Duis fermentum lacinia consequat. Donec id felis elit. Donec nec nunc felis. Morbi quis enim scelerisque, ullamcorper velit."
```

Comment:
I personally like the text, but not sure if customers will appreciate it. What about this:
```suggestion
                description="The page shows the collected product usage data. The current usage is computed from the last metrics received from sensors, and can be accurate to about 5 minutes. The maximum usage is aggregated hourly."
```